ingest/a_vs_b_de.py

```diff
@@ -8,13 +8,12 @@
 import scipy.stats
 import scanpy as sc
 import os
-
-
 import batchglm.api as glm
```

Comment:
If I recall, batchglm was added to see if using those parts of the tutorial would enable pairwise.summary to work. Since the library is not called in this work, we should remove this import for clarity.